frontend/src/index.tsx

```diff
@@ -3,18 +3,15 @@ import ReactDOM from "react-dom";
 import { Provider } from "react-redux";
 import { ThemeSwitcherProvider } from "react-css-theme-switcher";
 import store from "Src/store";
-import AppWrapper from "Src/modules/AppWrapper";
+import AppWrapper from "Src/AppRoutes";
```

Comment:
AppWrapper could be renamed to AppRouter

Reply:
Thanks updated